static/js/lib/courseAPI_test.js

```diff
@@ -0,0 +1,60 @@
+import { isLinkableCourseRun } from "./courseAPI"
+import { assert } from "chai"
+import moment from "moment"
+
+import { makeCourseRunEnrollment } from "../factories/course"
+
+describe("CourseRun is linkable or not", () => {
+  let userEnrollment
+  beforeEach(() => {
+    userEnrollment = makeCourseRunEnrollment()
+  })
+
```

Comment:
You can wrap all of the `it` blocks below in a `describe` block that indicates the API function being tested. This makes the test runner output more clear

```es6
  describe("isLinkableCourseRun", () => {
    it("", () => {})
    it("", () => {})
    //...
  })
```